src/cli/providers/messenger/attachment.js

```diff
@@ -0,0 +1,173 @@
+import fs from 'fs';
+import path from 'path';
+
+import invariant from 'invariant';
+import { MessengerClient } from 'messaging-api-messenger';
+import readdir from 'recursive-readdir';
+import fileType from 'file-type';
+import readChunk from 'read-chunk';
+import inquirer from 'inquirer';
+import jsonfile from 'jsonfile';
+import hasha from 'hasha';
+
+import getConfig from '../../shared/getConfig';
+import { error, warn, print, bold, log } from '../../shared/log';
+
+import help from './help';
+
+const getFileType = file => {
+  const imageType = ['jpg', 'png', 'jpeg', 'gif'];
+  const videoType = ['avi', 'mp4', 'm4v'];
+  const audioType = ['mp3', 'mid', 'm4a', 'wav'];
+
+  let type = 'file';
+
+  const buffer = readChunk.sync(file, 0, 4100);
+
+  const typeResult = fileType(buffer);
+  if (typeResult) {
+    const { ext } = typeResult;
+    if (imageType.includes(ext)) {
+      type = 'image';
+    } else if (videoType.includes(ext)) {
+      type = 'video';
+    } else if (audioType.includes(ext)) {
+      type = 'audio';
+    }
```

Comment:
Should return file type in else branch?

Reply:
the `type` is defined as 'file' at the beginning.
I've adjusted the order for readability.